src/ColumnEncodingUtility/analyze-schema-compression.py

```diff
@@ -220,10 +223,8 @@ def get_identity(adsrc):
     # checks if a column defined by adsrc (column from pg_attrdef) is
     # an identity, since both identities and defaults end up in this table
     # if is identity returns (seed, step); if not returns None
-    # TODO there ought be a better way than using a regex, especially since in theory
-    # someone could set a column to have a string that matches this regex as a default
```

Comment:
I thought I had a proof that it wasn't possible for this to match a string default, but I need to verify.